.github/README.md

```diff
@@ -30,6 +30,21 @@ Additionally, the OpenTelemetry Contrib collector has also been changed to the [
 3. Click "Create an API Key" to create one.
 4. Run `./demo.sh serverless docker`
 
+#### Connect to local cluster
+When you want to send data to an Elasticsearch instance running locally
+
+1. Create an API key
+```sh
+curl -X POST "http://localhost:9200/_security/api_key" -u elastic:changeme -H "Content-Type: application/json" -d'{ "name": "my_api_key" }'
```

Comment:
I would just user something like USER:PASSWORD here. Not everyone uses `elastic-package` or other tools where the default user:pass is `elastic:changeme`